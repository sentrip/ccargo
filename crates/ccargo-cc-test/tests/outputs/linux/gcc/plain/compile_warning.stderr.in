main.c: In function ‘main’:
main.c:4:14: warning: initialization of ‘int *’ from incompatible pointer type ‘long long int *’ [-Wincompatible-pointer-types]
    4 |     int* x = &v;
      |              ^
main.c:5:14: warning: initialization of ‘int *’ from incompatible pointer type ‘long long int *’ [-Wincompatible-pointer-types]
    5 |     int* y = &v;
      |              ^
