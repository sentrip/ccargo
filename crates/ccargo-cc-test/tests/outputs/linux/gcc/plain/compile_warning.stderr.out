warning: initialization of ‘int *’ from incompatible pointer type ‘long long int *’ [-Wincompatible-pointer-types]
   --> main.c:4:14 in function `‘main’`
    4 |     int* x = &v;
      |              ^
warning: initialization of ‘int *’ from incompatible pointer type ‘long long int *’ [-Wincompatible-pointer-types]
   --> main.c:5:14 in function `‘main’`
    5 |     int* y = &v;
      |              ^
